demos/gazoo.py

```diff
@@ -66,21 +100,44 @@ def parse_arguments():
     parser.add_argument("-n", "--num-cars", default=3, type=int,
                         help="The number of MOBIL cars on scene (default: 3).")
 
+    parser.add_argument("-m", "--maliput-backend", default="maliput_multilane", type=str,
+                        help="The maliput backend to use, maliput_osm or maliput_multilane (default: maliput_multilane).")
+
     return parser.parse_args()
 
 
-def create_gazoo_scenario_subtree(filename, mobil_cars_num):
+
+def create_gazoo_scenario_subtree(backend, mobil_cars_num):
+    LANES_MULTILANE = ["l:s1_0", "l:s1_1", "l:s1_2"]
+    LANES_MALIPUT_OSM = ["1825", "1405", "1352"]
+    filename = obtain_circuit_filepath(backend)
+
     # The road geometry
-    scenario_subtree = delphyne.behaviours.roads.Multilane(
-        file_path=filename,
-        name="circuit",
-    )
+    features = delphyne.roads.ObjFeatures()
+    features.draw_elevation_bounds = False
+    if(backend == "maliput_multilane"):
```

Comment:
Technical debt: we must change string based enums to be https://docs.python.org/3/library/enum.html